### PR TITLE
Rename CLI flags for v1

### PR DESCRIPTION
**Pull request by wemeetagain**
Implements https://www.notion.so/chainsafe/Lodestar-flags-02406e481f664d84adb56c2c348e49aa

with minimal diff

### CODE REVIEW

File: packages/cli/src/cmds/init/handler.ts [hunk 1]
```diff
@@ -52,15 +52,6 @@ export async function initializeOptionsAndConfig(args: IBeaconArgs & IGlobalArgs
...
   }
 
-  // Apply port option
-  if (args.port !== undefined) {
-    beaconNodeOptions.set({network: {localMultiaddrs: [`/ip4/0.0.0.0/tcp/${args.port}`]}});
-    const discoveryPort = args.discoveryPort ?? args.port;
-    beaconNodeOptions.set({network: {discv5: {bindAddr: `/ip4/0.0.0.0/udp/${discoveryPort}`}}});
-  } else if (args.discoveryPort !== undefined) {
-    beaconNodeOptions.set({network: {discv5: {bindAddr: `/ip4/0.0.0.0/udp/${args.discoveryPort}`}}});
-  }
```
Comment (dapplion) [inline packages/cli/src/cmds/init/handler.ts:62]: Handled in `packages/cli/src/options/beaconNodeOptions/network.ts` now

**Review by g11tech — APPROVED** [review]
LGTM! lets go :rocket:

**Review by dapplion — APPROVED** [review]
LGMT!

**Comment by dapplion** [comment]
@philknows Please could you write a migration guide for the flags changed when available?

**Comment by philknows** [comment]
> @philknows Please could you write a migration guide for the flags changed when available?

Adding this as part of the `v1.0.0` announcement.

**Pull request merged**